exwm-input.el

```diff
@@ -414,11 +415,21 @@ ARGS are additional arguments to CALLBACK."
                ;; It has been reported that the `window' may have be deleted
                (if (window-live-p window)
                    (select-window window)
-                 (setq window
-                       (get-buffer-window (exwm--id->buffer event) t))
+                 (setq window (get-buffer-window buffer t))
                  (when window (select-window window))))
-             ;; The event should be replayed
-             (setq mode xcb:Allow:ReplayPointer))))
+             (with-current-buffer buffer
+               (if (and (derived-mode-p 'exwm-mode)
+                          (eq exwm--on-KeyPress
+                              'exwm-input--on-KeyPress-line-mode)
+                          (setq read-event
+                                (exwm-input--mimic-read-event button-event))
+                          (exwm-input--forward-event-to-emacs-p read-event))
+                   (progn
+                     ;; The event should be forwarded to emacs
+                     (exwm-input--cache-event read-event)
+                     (exwm-input--unread-event button-event))
```

Comment:
`exwm-input--unread-event` should be defined before this so it would cause a compile warning.